mico-core/src/main/java/io/github/ust/mico/core/dto/KubernetesPodInformationDTO.java

```diff
@@ -0,0 +1,133 @@
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "PodName"),
+            @ExtensionProperty(name = "x-order", value = "10"),
+            @ExtensionProperty(name = "description", value = "Name of the Pod.")
+        }
+    )})
+    private String podName;
+
+    /**
+     * States the lifecycle of a pod: Pending, Running, Succeeded, Failed, Unknown, Completed, CrashLoopBackOff
+     */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "Phase"),
+            @ExtensionProperty(name = "x-order", value = "20"),
+            @ExtensionProperty(name = "description", value = "States the lifecycle of a pod: Pending, Running, Succeeded, Failed, Unknown, Completed, CrashLoopBackOff")
+        }
+    )})
+    private String phase;
+
+    /**
+     * IP address of the host to which the pod is assigned.
+     */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "HostIP"),
+            @ExtensionProperty(name = "x-order", value = "30"),
+            @ExtensionProperty(name = "description", value = "IP address of the host to which the pod is assigned.")
+        }
+    )})
+    private String hostIp;
+
+    /**
+     * Name of the node the pod is running on.
+     */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "NodeName"),
+            @ExtensionProperty(name = "x-order", value = "40"),
+            @ExtensionProperty(name = "description", value = "Name of the node the pod is running on.")
+        }
+    )})
+    private String nodeName;
+
+    /**
+     * Counter for restarts of all containers of this pod.
+     */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "Restarts"),
+            @ExtensionProperty(name = "x-order", value = "50"),
+            @ExtensionProperty(name = "description", value = "Counter for restarts of all container of a pod.")
+        }
+    )})
+    private int restarts;
+
+    /**
+     * Contains the start time of the pod.
+     */
+    @ApiModelProperty(extensions = {@Extension(
+        name = CustomOpenApiExtentionsPlugin.X_MICO_CUSTOM_EXTENSION,
+        properties = {
+            @ExtensionProperty(name = "title", value = "StartTime"),
+            @ExtensionProperty(name = "x-order", value = "51"),
+            @ExtensionProperty(name = "description", value = "The time when the pod was started.")
```

Comment:
Better: _The point in time the pod has been started_.

Reply:
For consistency, description property of API documentation and JavaDoc should match.